holoviews/plotting/plotly/dash.py

```diff
@@ -73,16 +73,18 @@ def get_layout_ranges(plot):
 def plot_to_figure(
         plot, reset_nclicks=0, layout_ranges=None, responsive=True, use_ranges=True
 ):
-    """
-    Convert a HoloViews plotly plot to a plotly.py Figure.
-
-    Args:
-        plot: A HoloViews plotly plot object
-        reset_nclicks: Number of times a reset button associated with the plot has been
-            clicked
-
-    Returns:
-        A plotly.py Figure
+    """Convert a HoloViews plotly plot to a plotly.py Figure.
+
+	Parameters
```

Comment:
All docstring in this file looks wrong.

Looks like tab/wrong indent issue.

Reply:
fixed